src/main/c/Makefile

```diff
@@ -239,6 +239,13 @@ freebsd64: export platform := freebsd/x86_64
 freebsd64:
 	$(MAKE) -f natives.mk
 
+# Requires a FreeBSD host running on aarch64 (arm64) architecture.
+freebsdarm64v8: export CFLAGS += -I$(JAVA_HOME)/include/freebsd -m64
+freebsdarm64v8: export platform := freebsd/ARM_64
+freebsdarm64v8: export variant := v8
```

Comment:
Ah, right, because `gnu.io.NativeResource.loadLib()` will always try to load a library called `libNRJavaSerialv8` for 64-bit ARM. Good catch.